src/SFA.DAS.EmployerIncentives.Commands.Types/ApprenticeshipIncentive/CreateIncentiveCommand.cs

```diff
@@ -1,37 +1,64 @@
-﻿using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+﻿using SFA.DAS.Common.Domain.Types;
+using SFA.DAS.EmployerIncentives.Abstractions.Commands;
 using SFA.DAS.EmployerIncentives.Abstractions.Logging;
 using SFA.DAS.EmployerIncentives.Domain.Accounts;
 using SFA.DAS.EmployerIncentives.Infrastructure.DistributedLock;
 using System;
+using System.Collections.Generic;
 
 namespace SFA.DAS.EmployerIncentives.Commands.Types.ApprenticeshipIncentive
 {
     public class CreateIncentiveCommand : DomainCommand, ILockIdentifier, ILogWriter
     {
-        public long AccountId { get; private set; }
-        public Guid IncentiveApplicationId { get; private set; }
-        public string LockId { get => $"{nameof(Account)}_{AccountId}"; }
+        public long AccountId { get; }
+        public long AccountLegalEntityId { get; }
+        public string LockId => $"{nameof(Account)}_{AccountId}";
+        public List<IncentiveApprenticeship> Apprenticeships { get; set; }
```

Comment:
Shouldn't there be one apprenticeship per command?  This command could be quite big.

Reply:
agreed